azurelinuxagent/ga/exthandlers.py

```
@@ -1467,9 +1484,13 @@ def download(self):
 
                 except PackageValidationError:
                     # validate_handler_manifest_signing_info() raises only ManifestValidationError, save_signature_validation_state()
-                    # raises only PackageValidationError. Both send logs/telemetry for any error, so we do nothing here.
+                    # raises only PackageValidationError. Both send logs/telemetry for any error. If signature is not being enforced,
+                    # add a message here that the error can be ignored.
                     # TODO: Raise error once signature validation is enforced.
-                    pass
+                    msg = "The signature validation failure can be ignored; will continue processing the extension."
```

msg should indicate the manifest validation failure can be ignored?

---

Updated to 'signingInfo' validation failure can be ignored